packages/cli/src/ucc.spec.ts

```diff
@@ -33,22 +35,20 @@ import {
   LatestGCPInfraConfigType
 } from "@opstrace/config";
 
-import {
-  LatestClusterConfigFileSchemaType
-} from "./schemas";
+import { LatestClusterConfigFileSchemaType } from "./schemas";
 
 import { uccGetAndValidate } from "./ucc";
 
 let tmpDir = "";
 
 beforeAll(() => {
   // create temp dir to store config files used in the tests
-  tmpDir = fs.mkdtempSync("ucctests")
+  tmpDir = fs.mkdtempSync("ucctests");
 });
 
 afterAll(() => {
   // cleanup the temp dir
-  fs.rmdirSync(tmpDir, { recursive: true });
+  fs.rmSync(tmpDir, { recursive: true });
```

Comment:
The previous variant had been deprecated for a long while, and was finally removed in NodeJS 16.